docs/user-guide/advanced-configuration/mesh-ingress.md

```diff
@@ -86,7 +86,7 @@ spec:
 !!! tip
     AWX Operator supports both API groups `traefik.io` and `traefik.containo.us` in `ingress_api_version` for Traefik, but it is recommended to use `traefik.io` since `traefik.containo.us` is deprecated in Traefik v2.10 and is removed in Traefik v3. Refer to [Traefik documentation](https://doc.traefik.io/traefik/migration/v2/#v210) for more information about deprecation.
 
-    If you can't see any IngressRouteTCP resources by `kubectl` command after deploying mesh ingress, you should fully qualify the resource name with API group, `kubectl get ingressroutetcp.traefik.io` or `kubectl get traefik.containo.us` for example.
+    If you can't see any IngressRouteTCP resources by `kubectl` command after deploying mesh ingress, you should fully qualify the resource name with API group, `kubectl get ingressroutetcp.traefik.io` or `kubectl get ingressroutetcp.traefik.containo.us` for example.
```

Comment:
@kurokobo The example in the TraefikLabs docs seems to be:

```
ingressroutetcps.traefik.io
```

https://doc.traefik.io/traefik/routing/providers/kubernetes-crd/

I also see: `ingressroutes.traefik.containo.us`

But I don't see an example of `ingressroutetcp.traefik.containo.us`.  Could you link to the docs for that?

Reply:
@rooftopcellist 
Hi, thanks for your confirmation.

> https://doc.traefik.io/traefik/routing/providers/kubernetes-crd/

This page contains both `ingressroutetcps.traefik.io` and `ingressroutetcps.traefik.containo.us` (search with plural form, `ingressroutetcps...` instead of `ingressroutetcp...`).

```bash
$ curl -s https://doc.traefik.io/traefik/routing/providers/kubernetes-crd/ | grep "name: ingressroutetcps"
  name: ingressroutetcps.traefik.io
  name: ingressroutetcps.traefik.containo.us
```